ocfweb/docs/docs/services/vhost.md

```diff
@@ -138,6 +138,14 @@ us informed if you have an existing OCF account).
 In other words, to comply with university policy, the OCF may only provide
 virtual hosting for websites that are hosted on OCF servers.
 
+* **Relevant domain name**: Student Organizations may only use domains or
```

Comment:
uncapitalize organizations? (or use the full name, Registered Student Organizations)